src/winscard_clnt.c

```diff
@@ -498,6 +491,40 @@ LONG SCardEstablishContext(DWORD dwScope, LPCVOID pvReserved1,
 	return rv;
 }
 
+static list_t contextMapList;
+
+CONSTRUCTOR static void constructor() {
+	int lrv;
+
+	lrv = list_init(&contextMapList);
+	if (lrv < 0)
+	{
+		Log2(PCSC_LOG_CRITICAL, "list_init failed with return value: %d",
+			lrv);
+		return SCARD_E_NO_MEMORY;
```

Comment:
You can't return an error code from a void function.

Reply:
f